clj-refactor.el

```diff
@@ -1511,7 +1511,7 @@ front of function literals and sets."
   (interactive)
   (insert "/")
   (when (and cljr-magic-requires
-             (looking-back cljr--magic-requires-re 6))
+             (looking-back (cljr--magic-requires-re) 6))
```

Comment:
It's used to limit the search to make it faster.  When this code was written I think the author meant it as "look back no more than 6 characters", but what it really means is "when looking back, look at the region between buffer position 6 and the position of point"